recipe-box-frontend/src/main/frontend/src/components/RecipesList/RecipeCard.tsx

```diff
@@ -53,7 +59,7 @@ export default function RecipeCard({
                 <h2>{title}</h2>
                 <p>{description}</p>
             </div>
-            <button className="view-recipe-button">VIEW RECIPE</button>
+            <button className="view-recipe-button" onClick={navigateToRecipeView}>VIEW RECIPE</button>
```

Comment:
You can use react router's Link component and set a button styling for it in order to navigate to the page without using the useNavigate hook. More details here:
https://reactrouter.com/en/main/start/tutorial#client-side-routing